test/rules/no-duplicate-super/test.ts.lint

```diff
@@ -0,0 +1,40 @@
+class A {
+    constructor() {
+        super();
+        super();
+        ~~~~~ [0]
+    }
+}
+
+class B {
+    constructor() {
+        super();
+        super.foo();
+    }
+}
+
+class C {
+    constructor() {
+        super();
+
+        class C2 {
+            constructor() {
+                super();
+            }
+        }
+    }
+}
+
+class D {
+    // Unfortunately, we warn here too.
+    constructor() {
+        if (cond()) {
+            super(true);
+        } else {
+            super(false);
+            ~~~~~ [0]
+        }
+    }
+}
+
+[0]: 'super()' is used twice. It must be called only once.
```

Comment:
would reword to something like: `multiple calls to super() found. It must be called only once.`